src/main/java/seedu/exceptions/ExceptionChecker.java

```diff
@@ -71,4 +72,13 @@ public static void checkAlreadyUnmark(Expenditure expenditure) throws AlreadyUnm
             }
         }
     }
+    public static void checkIfMoreThanTwoDecimalPlaces(String userInput, String DOT, String BLANK)
+            throws WrongPrecisionException, EmptyStringException {
+        if (userInput.contains(DOT)) {
+            String twoDecimalPlaces = ParseIndividualValue.parseIndividualValue(userInput,DOT,BLANK);
+            if (twoDecimalPlaces.length() > 2) {
```

Comment:
It may be better to name the an integer variable 2 such that it is even clearer that it is our desired precision

Reply:
We can implement this in the next push

Reply:
ah yeah good catch, will do so